### ISSUE TITLE
homebrew/mingw-w64: building curl fails on catalina

### ISSUE THREAD

**Issue by bassosimone** [measurement-kit/measurement-kit#1911]
```
=> cross-mingw x86_64 ./configure --disable-debug --disable-dependency-tracking --disable
Last 15 lines from /Users/sbs/Library/Logs/Homebrew/mingw-w64-curl/01.cross-mingw:
configure: PKG_CONFIG_LIBDIR will be set to "/usr/local/opt/mingw-w64-libressl/lib/pkgconfig"
checking for x86_64-w64-mingw32-pkg-config... no
checking for pkg-config... /usr/local/bin/pkg-config
configure: WARNING: using cross tools not prefixed with host triplet
checking for openssl options with pkg-config... found
configure: pkg-config: SSL_LIBS: "-lssl -lcrypto"
configure: pkg-config: SSL_LDFLAGS: "-L/usr/local/Cellar/mingw-w64-libressl/3.0.2/lib"
configure: pkg-config: SSL_CPPFLAGS: "-I/usr/local/Cellar/mingw-w64-libressl/3.0.2/include"
checking for HMAC_Update in -lcrypto... no
checking for HMAC_Init_ex in -lcrypto... no
checking OpenSSL linking with -ldl... no
checking OpenSSL linking with -ldl and -lpthread... no
configure: OPT_SSL: /usr/local/opt/mingw-w64-libressl
configure: OPENSSL_ENABLED: 
configure: error: --with-ssl was given but OpenSSL could not be detected
```

**Comment by bassosimone** [comment]
So, to avoid this issue I have released MK v0.10.11 that allows one to not link to libcurl. And now the issue appears again, but this time when configuring to build MK itself. Nice.

**Comment by bassosimone** [comment]
This problem has now clearly been fixed. Yet, the issue was larger than initially thought, as it always happens when we need to get our hands dirty with C/C++ code across several platforms.
